src/prepareData.js

```diff
@@ -18,7 +18,7 @@ export default function prepareData(renderProps, networkLayer) {
       environment,
       initialReadyState,
       queryAggregator,
-      render,
+      render: render(otherMiddleware),
```

Comment:
@KCraw Actually this line is not needed - when performing https://github.com/denvned/isomorphic-relay-router/commit/8ee27dda3092939656897332aa35b231b0db1fbf I just forgot to remove it. So there is no point to pass `otherMiddleware` to `prepareData`.